### ISSUE TITLE
What does "Center Position" really do?

### ISSUE THREAD

**Issue by GiannisGG** [opentoonz/opentoonz#1755]
While working on my cutout animation, I stumbled upon this problem. Try the following actions to recreate it:

#### 1\. Import an image into a level

It gets positioned at the centre (0,0) with 0º rotation angle.

#### 2\. Duplicate the level

Copy and Paste the whole level (let's call it *Level 1)* as a second, identical level (let's call it *Level 2).*

#### 3\. Level 1: Rotate its image as is

Using the **Animate Tool (A)** (ex-*Edit Tool,* I think), let's say you rotate the image at 30º.

The image revolves around its centre (0,0). Let's leave it there.

On top of it, of course, you can see the Level 2 image.

#### 4\. Level 2: Change the Center Position of its image

Again with the **Animate Tool (A)**, select Center from the drop down and choose **new coordinates for the rotation centre,** let's say (-50,-50).

Note that **no keyframe is being created by this change.** This means that moving the Center Position happens outside the keyframe logic. The question is *where?*

#### 5\. Level 2: Rotate its image around the new centre

Using the **Animate Tool,** rotate the image at 30º, as well.

The image revolves around its new centre (-50,-50).

Note that both images are now rotated by the same angle, but positioned offset.

#### 6\. Check the numbers

Open the **Function Editor** for the two levels.

According to the numbers, **the two levels look identical!**

#### 7\. Reset and compare

Level 1: reset the Rotation to 0.

Level 2: reset the Center Position to 0,0. Then, reset the Rotation to 0, as well.

The two images are not placed on the same position, but **still look identical in the Function Editor.**

#### What gives?

My guess is that the Center Position is **irrelevant to the keyframe logic,** but it somehow **affects the original position** of an imported image.

In other words, if you change the Center Position of an image, better stick to it, otherwise your animation messed up.

Anybody has more information on this subject?

Thanks.

**Comment by artisteacher** [comment]
From the Toonz Harlequin 7.1 User Guide:
_Note: The object center can be moved, but cannot change its position during the animation: once it is set, or modified, it is retained during all of the animation. If the center is changed many times, and you want to set it back to its original position, right-click the object in the schematic and choose Reset Center from the menu that opens_

Since the drawing center is not intended to be keyframed, it makes sense that its data is missing from the function editor. It is definitely important to make sure the drawing center is positioned properly before beginning any complex animations.

**Comment by RodneyBaker** [comment]
Good information but not a bug report.
Consider copying this information into the OpenToonz Forum.

Recommend Close.

**Comment by GiannisGG** [comment]
Thanks, @RodneyBaker. Do you mean the [Google group](https://groups.google.com/forum/#!forum/opentoonz_en)? Which category should I post it under?

**Comment by RodneyBaker** [comment]
No, OpenToonz has a bunch of channels under the primary OpenToonz discord.

This link *should* take you there:

https://discordapp.com/channels/313439701505933313/313439701505933313

**Comment by artisteacher** [comment]
There is a [related feature request](https://groups.google.com/forum/#!category-topic/opentoonz_en/feature-requests/bpgrhPf6W4U) in the forum already that you can contribute to.

**Comment by openingapps** [comment]
Just filed issue 4033, which reiterates the original complaint of this issue.  Two levels look like they have identical parameters, but they are drawn differently.  The new issue explains exactly what is hidden from you, and asks that it should not be hidden.
